weave/trace_server/clickhouse_schema.py

```diff
@@ -0,0 +1,107 @@
+import datetime
```

Comment:
moved from clickhouse_trace_server_batched. You can import there and token_costs, to keep ch definitions as 1 source of truth

Reply:
nice - good idea. There will likely be some merge conflicts soon here, just fyi